requirements.txt

```diff
@@ -13,3 +13,4 @@ virtualenv
 pluggage
 dockerstache>=0.0.9
 requests-toolbelt==0.6.2
+tox
```

Comment:
playing it fast and loose right now :) 
(they also do a very good job of keeping up with changes, so not so worried about this one, pay pin it at some point)